modules/apps/josh-lapurd-theme/src/WEB-INF/src/resources-importer/journal/templates/text_Grid_Paragraph_With_Header_1/text_Grid_Paragraph_With_Header_1.ftl

```diff
@@ -0,0 +1,4 @@
+<div class="grid-column-1 grid-text">
+    <h3>${grid_Header.getData()}</h3>
+    <p>${grid_Text.getData()}</p>
```

Comment:
```<p />``` tags are not needed since they can be added via the WYSIWYG editor.  If we need extra spacing, we should use a CSS class to achieve the spacing.